zk/stages/stage_sequence_execute_transactions.go

```diff
@@ -96,12 +99,23 @@ func extractTransactionsFromSlot(slot *types2.TxsRlp) ([]types.Transaction, []co
 		if err != nil {
 			// we have a transaction that cannot be decoded or a similar issue.  We don't want to handle
 			// this tx so just WARN about it and remove it from the pool and continue
-			log.Warn("Failed to decode transaction from pool, skipping and removing from pool", "error", err)
+			log.Warn("[extractTransaction] Failed to decode transaction from pool, skipping and removing from pool",
+				"error", err,
+				"id", slot.TxIds[idx])
 			toRemove = append(toRemove, slot.TxIds[idx])
 			continue
 		}
-		var sender common.Address
-		copy(sender[:], slot.Senders.At(idx))
+
+		// now attempt to recover the sender
+		sender, err := signer.SenderWithContext(cryptoContext, transaction)
```

Comment:
pprof shows that this line takes 27% of total time. I run sequencer and benchmark

Reply:
`sender, err := signer.SenderWithContext(cryptoContext, transaction)`

Reply:
https://github.com/okx/xlayer-erigon/issues/211

Reply:
Why attempt to recover sender from public key? there should be sender address in the transaction